core/src/main/java/bisq/core/support/dispute/Dispute.java

```diff
@@ -107,10 +132,25 @@
     @Setter
     @Nullable
     private String donationAddressOfDelayedPayoutTx;
+    // Added at v1.6.0
+    private Dispute.State disputeState = State.NEW;
+
+    // Should be only used in emergency case if we need to add data but do not want to break backward compatibility
+    // at the P2P network storage checks. The hash of the object will be used to verify if the data is valid. Any new
+    // field in a class would break that hash and therefore break the storage mechanism.
+    @Nullable
+    @Setter
+    private Map<String, String> extraDataMap;
+
     // We do not persist uid, it is only used by dispute agents to guarantee an uid.
     @Setter
     @Nullable
     private transient String uid;
+    @Setter
+    private transient long payoutTxConfirms = -1;
+
+    private transient final BooleanProperty isClosedProperty = new SimpleBooleanProperty();
+    private transient final IntegerProperty alertCountProperty = new SimpleIntegerProperty();
```

Comment:
🤔 not sure if alert count is the perfect naming. Maybe notificationCount or badgeCount would be more common for this?